scripts/build_android.sh

```diff
@@ -32,47 +32,54 @@ echo "Using configuration file: $CONFIG_FILE"
 # Read configurations from TOML file within [build] block
 DEVICE_TYPE=$(read_toml "$CONFIG_FILE" "build.android_device_type")
 BUILD_MODE=$(read_toml "$CONFIG_FILE" "build.build_mode")
+USE_WITNESS=$(read_toml "$CONFIG_FILE" "witness.use_witness")
 
 # Determine the architecture and folder based on device type
 case $DEVICE_TYPE in
     "x86_64")
         ARCHITECTURE="x86_64-linux-android"
         FOLDER="x86_64"
-        ;;
+    ;;
     "x86")
         ARCHITECTURE="i686-linux-android"
         FOLDER="x86"
-        ;;
+    ;;
     "arm")
         ARCHITECTURE="armv7-linux-androideabi"
         FOLDER="armeabi-v7a"
-        ;;
+    ;;
     "arm64")
         ARCHITECTURE="aarch64-linux-android"
         FOLDER="arm64-v8a"
-        ;;
+    ;;
     *)
         echo -e "${RED}Error: Invalid device type specified in config: $DEVICE_TYPE${DEFAULT}"
         exit 1
-        ;;
+    ;;
 esac
 
 # Determine the library directory and build command based on build mode
 case $BUILD_MODE in
     "debug")
         LIB_DIR="debug"
         COMMAND=""
-        ;;
+    ;;
     "release")
         LIB_DIR="release"
         COMMAND="--release"
-        ;;
+    ;;
     *)
         echo -e "${RED}Error: Invalid build mode specified in config: $BUILD_MODE${DEFAULT}"
         exit 1
-        ;;
+    ;;
 esac
 
+if [[ "$USE_WITNESS" == true ]]; then
```

Comment:
I find the name of this option confusing, we always use witness

USE_NATIVE_WITNESS or USE_CIRCOM_WITNESS_RS or something like this better

Reply:
[0aacfd6](https://github.com/oskarth/mopro/pull/86/commits/0aacfd671b1ea6cdb091b616aed1e43a722828f2)